src/program_options.hpp

```diff
@@ -217,6 +219,7 @@ class parser
       int const degree_in = NO_USER_VALUE, double const cfl_in = DEFAULT_CFL,
       bool const use_full_grid_in          = DEFAULT_USE_FG,
       int const max_level_in               = DEFAULT_MAX_LEVEL,
+      int const mixed_grid_group_in        = -1,
```

Comment:
```suggestion
      int const mixed_grid_group_in        = DEFAULT_MIXED_GRID_GROUP,
```
around line 185, add 
```
static auto constexpr DEFAULT_MIXED_GRID_GROUP = NO_USER_VALUE
```